sql/19_route.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE `route`
+(
+    `id`                 varchar(35)         NOT NULL,
+    `name`               varchar(100)        NOT NULL,
+    `description`        varchar(200)        NOT NULL,
```

Comment:
Are these limits correct? A UTF-8 string can have different length vs byte length. I don't think we need to be that aggressive?

Reply:
Actually, the in game max length for the name is 49 characters and the description is 239 characters. So which is suitable in SQL?

Reply:
UTF8 can take up to 4 bytes per character. Maybe make it 255 for both? MySQL will take only required space + 1 byte (up to 255 including).

https://dev.mysql.com/doc/refman/8.0/en/char.html

Reply:
So for name we can just use `varchar(255)`, and for description maybe just `varchar(65535)` or something?

Reply:
Another option is to not save descriptions at all.

Reply:
> So for name we can just use `varchar(255)`, and for description maybe just `varchar(65535)` or something?

You can't have strings that long in mariadb; and you want to avoid the combination of all fields going over a database page (well it won't let you anyway). The general accepted practice is to make them as big as you need and not much bigger ;-)

Reply:
> So for name we can just use `varchar(255)`, and for description maybe just `varchar(65535)` or something?

lol no. Just up to 255 you have lower cost of storing data, not stored data alone.

Reply:
Yeah I guess saving description up to 255 chars is also a reasonable alternative to not storing at all. :)